src/Components/BrainstormCategory/BrainstormCategory.tsx

```diff
@@ -18,16 +18,16 @@ const BrainstormCategory:React.FC<Props> = ({ category, formState, setCategory }
       setCategory({...formState, categories: [...formState.categories, category]})
     }
     if (!active) {
-      let newCategories = formState.categories.filter((cat:string) => cat !== category)
+      let newCategories = formState.categories.filter((ctg:Category) => ctg !== category)
       setCategory({...formState, categories: newCategories})
     }
   }
 
   useEffect(addRemoveCategory, [!active])
   let button; 
 
-  active ? button = (<button type='button' className='category-btn active' value={category} onClick={toggleActive}>{category}</button>)
-  : button = (<button type='button' className='category-btn' value={category} onClick={toggleActive}>{category}</button>)
+  active ? button = (<button type='button' className='category-btn active' value={category.name} onClick={toggleActive}>{category.name}</button>)
+  : button = (<button type='button' className='category-btn' value={category.name} onClick={toggleActive}>{category.name}</button>)
```

Comment:
I see you have almost the same buttons, one different thing is className. That's why you can make condition only for class, such as: 
```
const activeClass = active ? 'category-btn active' : 'category-btn';

// render
<button type='button' className={activeClass} value={category.name} onClick={toggleActive}>{category.name}</button>
```
It makes code less.

Reply:
Oh my goodness brilliant! Thank you!